README.md

```diff
@@ -251,14 +251,32 @@ editableAnnotationTypes: ["Ink", "Highlight"];
 
 The PSPDFKit React Native iOS Wrapper allows you to specify a custom grouping for the annotation creation toolbar. Please refer to [`RCTConvert+PSPDFAnnotationToolbarConfiguration.m`](./ios/RCTPSPDFKit/Converters/RCTConvert+PSPDFAnnotationToolbarConfiguration.m#L47) for the complete list of menu items. To set them just specify the `menuItemGrouping` prop on the `PSPDFKitView`. The format used is as follows:
 
-```
+```javascript
 [
   menuItem,
   { key: menuItem, items: [subItem, subItem]},
   ...
 ]
 ```
 
+#### Customize the Toolbar Buttons
+
+You can customize the toolbar buttons on the Native UI View component by specifying the toolbar buttons using `setLeftBarButtonItems` and `setRightBarButtonItems`, like so:
```

Comment:
Nitpick (feel free to ignore):Maybe we could add a sentence on what the second parameter is (the `animated` flag) and what it can be used for.

Reply:
I'd rather keep this paragraph short. The animated and the view mode parameters are documented in the API here:

https://github.com/PSPDFKit/react-native/blob/04aee962d12c61768efd2de431d6d27283705c19/index.js#L362-L381